website/src/components/props-table.tsx

```diff
@@ -50,6 +50,11 @@ const PropsTable = ({
     [info.props, omit, only],
   )
 
+  if (!entries.length) {
+    // this error breaks the build to notify you when there would be an empty table
+    throw new Error(`No props left to render for component ${of}`)
```

Comment:
Can you update this to explain how to resolve it? Maybe something like "Remove the use of `PropsTable` for this component".

Reply:
Update: 

https://github.com/chakra-ui/chakra-ui/blob/9b4ed1c804ce2bacef72a7640b4d90aa81623895/website/src/components/props-table.tsx#L53-L59

Reply:
NAILED IT!